From df4a018649f35d03ae7d197bcad701363546150b Mon Sep 17 00:00:00 2001
From: Niels Sascha Reedijk <niels.reedijk@gmail.com>
Date: Mon, 25 Dec 2023 21:34:58 +0000
Subject: [PATCH] Test commit for the format-checker

* Clock.rdef - change in file that is not going to be tested by clang-format
* cl_wind.cpp - two changes that will need to be reformated
* cl_view.cpp - rename a variable, file should already be in the correct format
* clock.h - no line changes (mode change only)

Change-Id: Ib68363538b702b8ce34dd3f48ea70a552d9e0afa
---
 src/apps/clock/Clock.rdef  |  2 +-
 src/apps/clock/cl_view.cpp | 20 ++++++++++----------
 src/apps/clock/cl_wind.cpp | 38 +++++++++++++++++++-------------------
 src/apps/clock/clock.h     |  0
 4 files changed, 30 insertions(+), 30 deletions(-)
 mode change 100644 => 100755 src/apps/clock/clock.h

diff --git a/src/apps/clock/Clock.rdef b/src/apps/clock/Clock.rdef
index 91d3a77f60..8b19c4047a 100644
--- a/src/apps/clock/Clock.rdef
+++ b/src/apps/clock/Clock.rdef
@@ -8 +8 @@ resource app_version {
-	middle = 0,
+	middle = 1,
diff --git a/src/apps/clock/cl_view.cpp b/src/apps/clock/cl_view.cpp
index 43c17ee28d..4829b462e2 100644
--- a/src/apps/clock/cl_view.cpp
+++ b/src/apps/clock/cl_view.cpp
@@ -49 +49 @@ dummy_label:
-		void* picH;
+		void* picHelp;
@@ -52 +52 @@ dummy_label:
-			if ((picH = rsrcs.FindResource('PICT', loop + 4, &len))) {
+			if ((picHelp = rsrcs.FindResource('PICT', loop + 4, &len))) {
@@ -54,2 +54,2 @@ dummy_label:
-				fClockFace[loop]->SetBits(picH, len, 0, B_CMAP8);
-				free(picH);
+				fClockFace[loop]->SetBits(picHelp, len, 0, B_CMAP8);
+				free(picHelp);
@@ -60 +60 @@ dummy_label:
-		if ((picH = rsrcs.FindResource(B_MINI_ICON_TYPE, "center", &len))) {
+		if ((picHelp = rsrcs.FindResource(B_MINI_ICON_TYPE, "center", &len))) {
@@ -62,2 +62,2 @@ dummy_label:
-			fCenter->SetBits(picH, len, 0, B_CMAP8);
-			free(picH);
+			fCenter->SetBits(picHelp, len, 0, B_CMAP8);
+			free(picHelp);
@@ -67 +67 @@ dummy_label:
-		if ((picH = rsrcs.FindResource('PICT', 13, &len))) {
+		if ((picHelp = rsrcs.FindResource('PICT', 13, &len))) {
@@ -69,2 +69,2 @@ dummy_label:
-			fInner->SetBits(picH, len, 0, B_CMAP8);
-			free(picH);
+			fInner->SetBits(picHelp, len, 0, B_CMAP8);
+			free(picHelp);
diff --git a/src/apps/clock/cl_wind.cpp b/src/apps/clock/cl_wind.cpp
index 6aa6b5f398..060bccef9c 100644
--- a/src/apps/clock/cl_wind.cpp
+++ b/src/apps/clock/cl_wind.cpp
@@ -25 +25 @@ TClockWindow::TClockWindow(BRect frame, const char* title)
-	_InitWindow();
+		_InitWindow();
@@ -37,12 +37,13 @@ TClockWindow::QuitRequested()
-	BPath path;
-	if (find_directory (B_USER_SETTINGS_DIRECTORY, &path, true) == B_OK) {
-		path.Append("Clock_settings");
-		int ref = creat(path.Path(), 0777);
-		if (ref >= 0) {
-			BPoint lefttop = Frame().LeftTop();
-			write(ref, (char *)&lefttop, sizeof(BPoint));
-			short face = fOnScreenView->ReturnFace();
-			write(ref, (char *)&face, sizeof(short));
-			bool seconds = fOnScreenView->ReturnSeconds();
-			write(ref, (char *)&seconds, sizeof(bool));
-			close(ref);
+		BPath path;
+		if (find_directory (B_USER_SETTINGS_DIRECTORY, &path, true) == B_OK) {
+			path.Append("Clock_settings");
+			int ref = creat(path.Path(), 0777);
+			if (ref >= 0) {
+				BPoint lefttop = Frame().LeftTop();
+				write(ref, (char *)&lefttop, sizeof(BPoint));
+				short face = fOnScreenView->ReturnFace();
+				write(ref, (char *)&face, sizeof(short));
+				bool seconds = fOnScreenView->ReturnSeconds();
+				write(ref, (char *)&seconds, sizeof(bool));
+				close(ref);
+			}
@@ -50,3 +51,2 @@ TClockWindow::QuitRequested()
-	}
-	be_app->PostMessage(B_QUIT_REQUESTED);
-	return true;
+		be_app->PostMessage(B_QUIT_REQUESTED);
+		return true;
@@ -61 +61 @@ TClockWindow::_InitWindow()
-	
+
@@ -83 +83 @@ TClockWindow::_InitWindow()
-			
+
@@ -85 +85 @@ TClockWindow::_InitWindow()
- 
+
diff --git a/src/apps/clock/clock.h b/src/apps/clock/clock.h
old mode 100644
new mode 100755
-- 
2.39.2

